client/src/app/access/logout/logout.component.ts

```diff
@@ -0,0 +1,24 @@
+import { DOCUMENT } from '@angular/common';
```

Comment:
When was this added 🤔?

I always used to create my own wrappers around `window` and `document` globals.

Reply:
Not sure! Most of this copde / functionality is from https://auth0.github.io/auth0-angular/index.html#getting-started